lib/wakes/redirect_mapper.rb

```diff
@@ -33,11 +35,11 @@ def resources_affected
     end
 
     def target_location
-      @target_location ||= Wakes::Location.find_by(:path => target)
+      @target_location ||= Wakes::Location.find_by(:host => host(target_path), :path => path(target_path))
```

Comment:
@chrismaddocks wouldn't it be simpler to just setup `target_host` and `source_host` variables in the `#initialize` method? ALSO, doesn't `path(target_path)` seem wrong to you? If it says `target_path`, it should be the path!

You could also do something like the following:

    def initialize(source_path_or_url, target_path_or_url, label = nil)
      @source_path_or_url = source_path_or_url
      @target_path_or_url = target_path_or_url
      ...
    end

    def source_path
      path(@source_path_or_url)
    end

    def source_host
      host(@source_path_or_url)
    end

    def target_path
      ...